taichi/backends/vulkan/aot_module_loader_impl.cpp

```diff
@@ -69,6 +94,26 @@ class AotModuleImpl : public aot::Module {
   }
 
  private:
+  std::unique_ptr<aot::Field> make_new_field(const std::string &name) override {
+    aot::CompiledFieldData field;
```

Comment:
shall we put this directly in the `get_field` function?

Reply:
sure